### PR TITLE
Update ja.js

### PR DESCRIPTION
**Pull request by jins-tkomoda**
Add Japanese dictionary translations. (sorry, i couldn't update label to "type: translation",,, )

### CODE REVIEW

File: lib/locales/ja.js [hunk 1]
```diff
@@ -11,7 +11,64 @@
...
+        'Zoom in': '拡大',                                                                                  // components/modebar/buttons.js:121
+        'Zoom out': '縮小',                                                                                 // components/modebar/buttons.js:130
+        'close:': '閉じる:',                                                                                // traces/ohlc/transform.js:139
+        'trace': 'トレース:',                                                                               // plots/plots.js:439
+        'lat:': '緯度:',                                                                                    // traces/scattergeo/calc.js:48
+        'lon:': '経度:',                                                                                    // traces/scattergeo/calc.js:49
+        'q1:': 'q1:',                                                                                       // traces/box/calc.js:130
+        'q3:': 'q3:',                                                                                       // traces/box/calc.js:131
```
Comment (alexcjohnson) [inline lib/locales/ja.js]: Are q1 and q3 OK left in English? These are the first and third quartiles, in a box plot.

**Comment by alexcjohnson** [comment]
Thanks @jins-tkomoda - can you recommend another Japanese speaker to review?

**Comment by alexcjohnson** [comment]
The two newest keys (which some of our first translations haven't updated to add) didn't make it into your list, could you add them? These are:
["Click to enter radial axis title"](https://github.com/plotly/plotly.js/blob/7d91d29a48bc882afccda635054dfa1a7ece85fd/dist/translation-keys.txt#L10)
["new text"](https://github.com/plotly/plotly.js/blob/7d91d29a48bc882afccda635054dfa1a7ece85fd/dist/translation-keys.txt#L50)

**Comment by jins-tkomoda** [comment]
@yuji-uema @mbfts-ts could you check the translations?

**Pull request merged**